src/content/structured/get-started/react.mdx

```diff
@@ -136,3 +136,15 @@ Replace any instances of `<svg>` with `<SlottedSVG>`.
   </SlottedSVG>
 </IcTopNavigation>
 ```
+
+### Step three
```

Comment:
Adding MUI Icons isn't really an additional step to get started with our React component. It's more of a 'If you want to use MUI Icons' section. The section could be called 'Using MUI Icons with SlottedSVG'

Reply:
*Using Material Design Icons (MDI) with SlottedSVG